wpeview/src/main/cpp/Runtime/WKWebView.cpp

```diff
@@ -575,6 +745,33 @@ void JNIWKWebViewCache::nativeScriptDialogConfirm(
     webkit_script_dialog_confirm_set_confirmed(dialog, static_cast<gboolean>(confirm));
 }
 
+void JNIWKWebViewCache::nativeSetTLSErrorsPolicy(
```

Comment:
Same as with the other setting above, I am not a fan of attaching methods to the web view class that modify options that can affect other web views.

Reply:
In this case I think it is only impacting the web view because it applies to the network session associated with the web view. At least, I understand that the network session is private to the webview, or maybe I'm wrong. If it is not private to the web view, then the "load-failed-with-tls-errors" signal should be on the network session and not on the webview itself.

To be honest I'm a bit confused here because I was used to the WebSiteDataManager approach before. I'm not really sure how the NetworkSession is behaving exactly here.

Reply:
Sorry, I've confounded `JNIWKWebViewCache::nativeSetTLSErrorsPolicy` with `JNIWKWebViewCache::nativeTriggerSslErrorHandler`.

For  `JNIWKWebViewCache::nativeSetTLSErrorsPolicy` it should be logical to have it into `WKNetworkSession.cpp`, this is what I did first but it was not working as `webkit_web_view_get_network_session(wkWebView->webView())` was returning a network session than the one wrapped into ``WKNetworkSession.cpp`.

There must be a bug somewhere, so I'm adding a FIXME to check it deeper later.